### DISCUSSION TITLE
Serial Port Connections in Windows 11

### DISCUSSION THREAD

**Discussion by IrisEmbedded-DM** [Serial-Studio/Serial-Studio#123] [General]
I'm currently testing out Serial Studio for visualising data from my embedded sensors, running on Windows 11. My COM port displays in the list, but when I connect, the data only displays maybe once every 20 times I launch the software. I've pared the data back to be as simple as possible (just CSV with a newline) and yet this still rarely displays in the console. I have also tried setting up frames and the JSON, but this always leads to no data.

Checking the COM port with other software shows that the data is there! 
![ss1](https://user-images.githubusercontent.com/77390280/188437079-3885733c-2ffc-4569-acad-419b5a2486a8.png)
![ss2](https://user-images.githubusercontent.com/77390280/188437081-5eb5d388-aa37-4b99-9f3d-7187a9491ef4.png)

**Comment by alex-spataru** [comment]
Thanks for @recursivenomad for adding a possible fix for this issue. The latest build adds a checkbox that allows users to send the DTR (Data Terminal Ready) signal upon connecting to a serial port device:

<img width="1204" alt="Screenshot 2024-08-05 at 3 40 40 p m" src="https://github.com/user-attachments/assets/9e9a3019-3bf8-4ec6-8a4d-5c14c73b71b9">

Please let me know if the latest [continuous](https://github.com/Serial-Studio/Serial-Studio/releases/tag/continuous) build fixes this issue!
